manifests/concourse-manifest/concourse-base.yml

```diff
@@ -73,21 +73,22 @@ jobs:
           databases:
             - name: atc
               role: atc
-              password: dummy-password
+              password: (( grab secrets.concourse_postgres_password ))
 
       - name: atc
         release: concourse
         properties:
           external_url: (( concat "https://" terraform_outputs.concourse_dns_name ))
           basic_auth_username: admin
           basic_auth_password: (( grab secrets.concourse_atc_password ))
+          auth_duration: (( grab $CONCOURSE_AUTH_DURATION ))
```

Comment:
I recommend you to use `auth_duration: (( grab $CONCOURSE_AUTH_DURATION || null ))` or, otherwise, add a fixture and change the rspec of the manifest generation.